### PR TITLE
Styles for Draggable Button, Draggable Card, and Draggable Target

### PR DESCRIPTION
**Pull request by juligobbi-fluxit**
Added the style of the different states as default, hover, active, and dragging to the 3 components of the draggable flow to accompany the real interaction of the movement: Draggable Button, Draggable Card, and Draggable Target. 
Also fixes for square corners on bug chrome

### CODE REVIEW

File: src/components/draggable/draggableTarget.tsx [hunk 1]
```diff
@@ -16,7 +16,14 @@ const DraggableTarget: React.FC<DraggableTargetProps> = ({
         onCollect,
     });
 
-    const actualCss = draggableTarget(undefined, undefined, isOver, canDrop);
+    const actualCss = draggableTarget(
+        undefined,
+        undefined,
+        undefined,
+        undefined,
```
Comment (BrandonDenham) [inline src/components/draggable/draggableTarget.tsx:23]: as these are always undefined, they shouldn't be function parameters

Comment (BrandonDenham) [reply src/components/draggable/draggableTarget.tsx:23]: just do it internally inside the function, don't add parameters that won't be used, thats leaking implementation details

**Review by BrandonDenham — APPROVED** [review]
please address the comment, but no need for re-review

File: src/components/draggable/draggableTarget.tsx (hunk 1 above)
Comment (juligobbi-fluxit) [reply src/components/draggable/draggableTarget.tsx]: You are right, I agree. We have changed the parameters with Eze in the following commit

**Pull request merged**